### DISCUSSION TITLE
Shutting down the proxy server

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#70] [Announcements]
Hello, We would like to announce that we have now stopped providing free cors to all users sadly due to server overload that is caused by the proxy server. Do not worry we are still looking for another server that allows us to do it but for now, all streams will return errors for all users. 

### Setting up your own proxy
To set or remove the proxy servers, Click on the settings, then find the proxy URL textbox and remove the default proxy URL, or use your own proxy server after that click __Save Proxy URL__ then refresh the page once again to apply the change.

![image](https://github.com/user-attachments/assets/4b4907dc-c18b-401f-a9ff-f1c4f152afef)

### Use a browser extension

We recommended this extension and it works on almost every PC user.
- Chromium: https://chromewebstore.google.com/detail/cors-unblock/lfhmikememgdcahcdlaciloancbhjino
- Firefox: https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere
- Opera: https://addons.opera.com/en/extensions/details/cors-toggle
- Edge: https://microsoftedge.microsoft.com/addons/detail/cors-unblock/hkjklmhkbkdhlgnnfbbcihcajofmjgbh

**Comment by rishabnotfound** [comment]
Tried sir but it still didnt worked

**Reply by NYT92** [reply]
What error are you facing right now?
